bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway/Gateway.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Gateway
+{
+    /// <summary>
+    ///  Wrapper for Azure IoT Gateway SDK. 
+    /// </summary>
+    public class Gateway
```

Comment:
So we have a class named `Microsoft.Azure.Devices.Gateway.Gateway`. I loathe the redundancy, but I'm not sure what a better name would be.  :(  Do you have any ideas?